### PR TITLE
Refactor for Waken(?)

### PR DESCRIPTION
**Pull request by nkalupahana**
This PR re-factors Witness to have three dashboards: the hacker dashboard, the judge dashboard, and the organizer dashboard. This makes it so you don't have to conditionally render on every user type in every component -- instead, the dashboard has what's allowed to be shown.

This PR also separates some stuff into different component files, and standardizes some types/etc.

### CODE REVIEW

File: pages/OrganizerDash.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		}
+		return (await res.json()) as ScheduleDisplay[];
+	});
+    
+    return (
+        <>
+			{ scheduleData && <OrganizerSchedule data={scheduleData} /> }
+			<Divider />
+            { teamsData && <>
+				{ /* Add dropdown here w/functionlity */ }
```
Comment (aadibajpai) [inline pages/OrganizerDash.tsx:56]: typo
```suggestion
				{ /* Add dropdown here w/ functionality */ }
```

**Comment by aadibajpai** [comment]
@nkalupahana conflict

**Comment by aadibajpai** [comment]
image
should I be seeing more as of now

**Comment by aadibajpai** [comment]
also, @nkalupahana a general summary of changes and reasoning would be really good as we move into restructuring code so everyone has a better idea of what has changed moving forward.

**Comment by nkalupahana** [comment]
Ready for review again @aadibajpai

File: lib/mongodb.ts [hunk 2]
```diff
@@ -2,7 +2,10 @@
 import { MongoClient } from 'mongodb';
 
 const uri = process.env.DATABASE_URL;
-const options = {};
+const options = {
+	useUnifiedTopology: true,
+	useNewUrlParser: true,
+};
```
Comment (Ace314159) [inline lib/mongodb.ts:9]:
```suggestion
const options = {};
```

**Pull request merged**